src/app/shared/components/template/components/layout/display-group/display-group.component.html

```diff
@@ -3,7 +3,11 @@
   @if (params.sticky === "top") {
     <div [style.height.px]="sticky.height()"></div>
   }
-  <tmpl-display-group-sticky [position]="params.sticky" #sticky>
+  <tmpl-display-group-sticky
+    [position]="params.sticky"
+    #sticky
+    [attr.data-param-style]="params.style"
```

Comment:
Because there is an additional layer of nesting when rendering a sticky display group, we need to re-apply the style params here in order to make them available as selectors on the `:host` element within the sticky display group component.